lib/reservation.rb

```diff
@@ -0,0 +1,29 @@
+require 'date'
+require 'pry'
+
+class Reservation
+
+  attr_reader :start_date, :end_date, :room
+
+  #receives the start_date, end_date, and room info from ReservationMaker
+  def initialize (start_date, end_date, room)
+    @start_date = start_date
+    @end_date = end_date
+    @room = room
+    @reservation_duration = reservation_duration
```

Comment:
besides here in the definition, `@reservation_duration` is only called within `Reservation` once, in the `cost` method... does `@reservation_duration` really need to be stored, or can the one instance that it's called (again, in the `cost` method) just recalculate this value by calling the method?

aka do we need this instance variable?